scripts/get-access-key.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -x
+
+function getAccessKey() {
+
+echo "Generate Admin Access Key" > /dev/tty
+
+COOKIES=$(curl -s -c - "${ARTIFACTORY_URL}/ui/api/v1/ui/auth/login?_spring_security_remember_me=false" \
```

Comment:
These are bind variables and can escape the function

Reply:
removed